### PR TITLE
Fix README external links (#1)

### PR DESCRIPTION
**Pull request by ovidiubute**
Fix a few typos in Markdown links.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ store.dispatch({ type: "INCREMENT" });
 console.log(store.getState().count); // => 1
 ```
 
-`create-state` is a lightweight implementation of a [Redux][] store without advanced concepts like middleware.
+`create-state` is a lightweight implementation of a [Redux](https://redux.js.org/introduction) store without advanced concepts like middleware.
```
Comment (philipp-spiess) [inline README.md]: This works as expected and is using [reference-style links](https://daringfireball.net/projects/markdown/syntax#link). 🙂 

```suggestion
`create-state` is a lightweight implementation of a [Redux][] store without advanced concepts like middleware.
```

File: README.md [hunk 2]
```diff
@@ -63,4 +63,4 @@ Every help on this project is greatly appreciated. To get you started, here's a
 
 [MIT](https://github.com/philipp-spiess/create-store/blob/master/README.md)
 
-[Redux]: https://redux.js.org/introduction
+[Redux](https://redux.js.org/introduction)
```
Comment (philipp-spiess) [inline README.md]: This is part of the [reference-style link](https://daringfireball.net/projects/markdown/syntax#link) from above.

```suggestion
[Redux]: https://redux.js.org/introduction
```

**Review by philipp-spiess — APPROVED** [review]
Thanks for spotting this! Can you revert the changes to the reference-style link?

**Comment by ovidiubute** [comment]
@philipp-spiess thanks, didn't know about reference-style links 😃

**Review by philipp-spiess — APPROVED** [review]
Thanks again 😊

**Pull request merged**